pcsx2/GS/GS.h

```diff
@@ -33,8 +33,9 @@
 #include <d3dcompiler.h>
 #include <d3d11_1.h>
 #include <dxgi1_3.h>
-#include <comutil.h>
-#include <atlcomcli.h>
+
+#include "wil/com.h"
+#include "wil/resource.h"
```

Comment:
```suggestion
#include <wil/com.h>
#include <wil/resource.h>
```

Additionally with this one I'd prefer if we didn't put it in the `GS.h` header but rather individual headers/cpp files that depend on it.

see #4709